日本語への翻訳: Not Enough Crashes

fltuna opened the pull request:
このプルリクエストでは、Not Enough Crashesの日本語訳を追加します。

校閲は行いましたが、不自然な点があればご指摘いただければ修正します。

fltuna committed:
Add ja_jp.json

fltuna committed:
Proofread ja_jp.json

fltuna commented on kubejs/assets/notenoughcrashes/lang/ja_jp.json:
```diff
@@ -0,0 +1,39 @@
+  "notenoughcrashes.gui.getLink": "リンクを入手",
+  "notenoughcrashes.gui.uploadToCrashy": "Crashyで表示",
+  "notenoughcrashes.gui.loadingCrashyUpload": "アップロード中...",
+  "notenoughcrashes.gui.failed": "[失敗しました。 NECにレポートを送信してください]",
```
Line 5の原文が "[Failed, Report to NEC]"となっているのですがNECだとぱっと見でNot Enough Crashesだとわからない可能性があり、原文を尊重し"NEC"のままにするか、それともわかりやすいように"Not Enough Crashes"にするか迷っているのですがどちらが良いでしょうか？

AIMURASAN replied:
"失敗しました。 'NEC' にレポートを送信してください"
でいいとおもう

AIMURASAN replied:
他Modでは、同時に下線URLも書いてあったり、なかったり

SOL3675 replied:
自分でModを導入している人はいいけど、Modpackとして遊んでいる人にとって "NEC" のままだと、何のことかわからず、某企業？ってなりそうなので、"Not Enough Crashes"の方がいいかな。
多分、原文こそ直されるべきな気がする。

SOL3675 replied:
ボタンになっているとかで表示幅が足りないのならNECのままでもいいかも。

fltuna replied:
今ソースコードを確認したのですが、恐らくボタンでの実装っぽいのでNECのままにしたいと思います。

https://github.com/natanfudge/Not-Enough-Crashes/blob/1.19/common/src/main/java/fudge/notenoughcrashes/gui/ProblemScreen.java#L129

fltuna replied:
ボタンでの実装であると見受けられるため、変更無しとし解決済みとします。

fltuna commented on kubejs/assets/notenoughcrashes/lang/ja_jp.json:
```diff
@@ -0,0 +1,39 @@
+  "notenoughcrashes.midnightconfig.catchInitializationCrashesComment2": "特別なクラッシュ画面は表示されません。",
+  "notenoughcrashes.midnightconfig.catchInitializationCrashes": "初期化中のクラッシュを検出",
+  "notenoughcrashes.midnightconfig.debugModIdentificationComment": "trueの場合、開発者のためにより詳細な情報がログに記録されます。",
+  "notenoughcrashes.midnightconfig.debugModIdentification": "デバッグ用MODの識別",
```
Line 35の原文"Debug Mod Identification"はModの識別情報をデバッグ用にログに記録するのか、デバッグ用のModを識別してログに記録するのかよくわからず"デバッグ用MODの識別"としたのですがどれが正しいのでしょうか...

SOL3675 replied:
これはどのバージョンのNot Enough Crashesに対応してる？
28行目以降は 1.19.2 にはないはず。
英語はこれ。
https://github.com/natanfudge/Not-Enough-Crashes/blob/1.19/common/src/main/resources/assets/notenoughcrashes/lang/en_us.json

fltuna replied:
今確認しました、1.19.3ブランチで追加された項目でした。
https://github.com/natanfudge/Not-Enough-Crashes/commit/2ae5fb0f649ba5217685bc4b6e5e2be2a120c639#diff-6657003e8c7819ba2c4f68e9cfb673a7c62b1fed1943cb20934d0294a8b9a834

1.19ブランチの方で作り直します。

たまたま、私の方でNot Enough Crashesの翻訳をしている最中で、それを持ってきたためバージョンの確認をし忘れていました。
https://github.com/faketunaPrivateCamp/Not-Enough-Crashes

fltuna replied:
修正が完了したので、解決済みとします。

fltuna commented on kubejs/assets/notenoughcrashes/lang/ja_jp.json:
```diff
@@ -0,0 +1,39 @@
+  "notenoughcrashes.crashscreen.paragraph3.line4": "インストールされているためクラッシュしてもプレイを続行できます。",
+  "notenoughcrashes.initerrorscreen.title": "Minecraftの起動に失敗しました!",
+  "notenoughcrashes.initerrorscreen.summary": "起動時にエラーが発生したためMinecraftを起動することができませんでした。",
+  "notenoughcrashes.initerrorscreen.paragraph3.line1": "\"リンクを入手\" をクリックするとブラウザーが開きます。",
```
Line 24に"ボタン"の文言を入れ忘れたので修正します。

fltuna replied:
修正済み

fltuna committed:
Fix ja_jp.json

fltuna committed:
Fix ja_jp.json
元の翻訳ファイルが1.19.3ブランチのものであり、今回のターゲットである1.19ブランチから取得していなかったため
1.19ブランチと同じ翻訳キーのみを残した。

fltuna commented:
懸念点はすべて解決としました。
その他にご不明な点があればご指摘ください。
